bindings/python/examples/burn_native_tokens.py

```diff
@@ -0,0 +1,24 @@
+from iota_wallet import IotaWallet
+import time
```

Comment:
You import this in all examples, but it's only used in the last one?

Reply:
good eyes